### PR TITLE
GSMP: New Execution Mode 'J'

### PR DESCRIPTION
**Pull request by mseich**

### CODE REVIEW

File: file-formats/gsmp/type/zif_aff_gsmp_v1.intf.abap [hunk 1]
```diff
@@ -101,6 +101,10 @@ INTERFACE zif_aff_gsmp_v1
       "! No execution. Only model. Data from external.
       none        TYPE ty_execution_mode VALUE 'N',
 
+      "! <p class="shorttext">Job</p>
+      "! Job.
```
Comment (schneidermic0) [inline file-formats/gsmp/type/zif_aff_gsmp_v1.intf.abap:105]: I think, I would remove the '.' at the end

```suggestion
      "! Job
```

File: file-formats/gsmp/gsmp-v1.json [hunk 2]
```diff
@@ -102,19 +102,22 @@
...
+            "None",
+            "Job"
           ],
           "enumDescriptions": [
             "System",
             "Instance",
             "Application",
-            "No execution. Only model. Data from external."
+            "No execution. Only model. Data from external.",
+            "Job."
```
Comment (schneidermic0) [inline file-formats/gsmp/gsmp-v1.json:120]:
```suggestion
            "Job"
```

File: file-formats/gsmp/type/zif_aff_gsmp_v1.intf.abap [hunk 3]
```diff
@@ -101,6 +101,10 @@ INTERFACE zif_aff_gsmp_v1
...
+      job        TYPE ty_execution_mode VALUE 'J',
+
```
Comment (schneidermic0) [inline file-formats/gsmp/type/zif_aff_gsmp_v1.intf.abap:107]: Minor: Would it make sense to move `job` before `none` so that `none` is the last entry?

**Review by schneidermic0 — CHANGES_REQUESTED** [review]
Just some minor suggestions for improvement

**Comment by schneidermic0** [comment]
fixes #379.

**Review by schneidermic0 — APPROVED** [review]
Looks good to me.

Did you also see [this question](https://github.com/SAP/abap-file-formats/pull/382#discussion_r928875852)?

File: file-formats/gsmp/type/zif_aff_gsmp_v1.intf.abap (hunk 3 above)
Comment (mseich) [reply file-formats/gsmp/type/zif_aff_gsmp_v1.intf.abap]: Wird heute Mittag korrigiert

Comment (mseich) [reply file-formats/gsmp/type/zif_aff_gsmp_v1.intf.abap]: done ...

**Review by schneidermic0 — APPROVED** [review]
Thanks, looks good to me.

**Pull request merged**